examples/gcs/README.md

```diff
@@ -20,9 +21,33 @@ resources:
     class: basic-admin
 ```
 
+## Infrastructure setup
+The workload service account will be automatically assigned to the necessary role with the selected policy.
+
+```mermaid
+graph TD;
+  gcs["Google Cloud Storage"]
+  policy["Google Cloud IAM Policy"]
+  role["Google Cloud IAM Role"]
+  subgraph EKS Cluster
+    pod[workload pod]
+    service[Service Account]
+  end
+  policy --> gcs
+  policy --> role --> service --> pod
```

Comment:
I think this isn't correct, in GCP we "bind" a member (the service account) a role on a specific entity (the GCS bucket).

So I think it would be more correct to point the arrow from the ServiceAccount ~> GCS and write bind "role" on the arrow itself.

Reply:
Done!